swagger.yaml

```
@@ -4526,6 +4743,10 @@ components:
           type: integer
           example: 0
           description: Count of asset mints and burns within the transaction
+        redeemer_count:
+          type: integer
+          example: 0
+          description: Count of redemmers within the transaction
       required:
         - hash
         - block
```

```suggestion
      required:
        - redeemer_count
        - hash
        - block
```

`redeemer_count` is missing from required properties.